src/loki/search.cc

```diff
@@ -514,14 +519,22 @@ struct bin_handler_t {
         continue;
       }
 
+      // lots of places below where we might like to know about the opp edge
+      const DirectedEdge* opp_edge = nullptr;
+      graph_tile_ptr opp_tile = tile;
+      GraphId opp_edgeid;
```

Comment:
there are now 3 different places where we fallback to testing the opposing edge, one for `allowed`, one for `search_filter` and one for `reach` check. it was messy but i pulled the declaration of these up here so i could use the same ones for everything.

Reply:
would be nice if there were an easy way to memoize the getting of the opposing triplet of variables